package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-flip-move",
-  "version": "2.9.14",
```

Comment:
Guess I never pushed the `2.9.15` branch, my bad. That was the release for zombie-nodes bugfix.